addons/account/models/account_move.py

```diff
@@ -117,6 +117,21 @@ def _sequence_fixed_regex(self):
         tracking=True,
         default='draft',
     )
+
+    visualization_state = fields.Selection(
+        selection=[
+            ('draft', 'Draft'),
+            ('posted', 'Posted'),
+            ('sent', 'Sent'),
+            ('cancel', 'Cancelled'),
+        ],
+        required=True,
+        readonly=True,
+        copy=False,
+        tracking=True,
+        compute='_compute_visualization_state'
```

Comment:
(we could add a comma at the end so that if one day people want to modify the field we keep the github history)